apps/docs/.vitepress/config.ts

```diff
@@ -57,8 +57,8 @@ export default defineConfig({
         ],
       },
       {
-        text: 'Basics',
-        link: '/guide/basics/',
+        text: 'Guide',
```

Comment:
To be as similar as possible to the docs hub, perhaps we should remove all sections, including `Introduction` and `Guide`, and have a flat list.

Reply:
Could all items look like the `Getting Started` and `Creating a Fuel dApp` items?

Reply:
Unrelated to this PR, just cross-referencing these here:
 - https://github.com/FuelLabs/fuels-ts/issues/2086
 - https://github.com/FuelLabs/fuels-ts/issues/2087